packaging/linux/amazon-cloudwatch-agent.spec

```diff
@@ -10,6 +10,8 @@ Source:     amazon-cloudwatch-agent.tar.gz
 
 %define _enable_debug_packages 0
 %define debug_package %{nil}
+%define _source_payload w6.gzdio
+%define _binary_payload w6.gzdio
```

Comment:
overview still says gzip 0.

After evaluating xz compression and gzip compression with different compression levels we are choosing level 6 (which is the default)

Current binary size is X compared to before it was Y.
Current install time is X compared to before it was Y.

Reply:
Done